src/frontend/src/optimizer/plan_node/logical_agg.rs

```diff
@@ -532,6 +532,19 @@ impl LogicalAggBuilder {
         Ok(())
     }
 
+    /// Push a new planned agg call into the builder.
+    /// Return an `InputRef` to that agg call.
+    /// For existing agg calls, return an `InputRef` to the existing one.
+    fn push_agg_call(&mut self, agg_call: PlanAggCall) -> InputRef {
+        if let Some((pos, existing)) = self.agg_calls.iter().find_position(|&c| c == &agg_call) {
+            return InputRef::new(self.group_key.len() + pos, existing.return_type.clone());
+        }
+        let index = self.group_key.len() + self.agg_calls.len();
```

Comment:
```suggestion
        if let Some((pos, existing)) = self.agg_calls.iter().find_position(|&c| c == &agg_call) {
            return InputRef::new(self.agg_start_offset() + pos, existing.return_type.clone());
        }
        let index = self.group_key.len() + self.agg_calls.len();
```

Minor suggestion for readability.